src/QuartzHostedService/IServiceCollectionExtensions.cs

```diff
@@ -26,11 +28,29 @@ public static IServiceCollection AddQuartzHostedService(this IServiceCollection
             var re = services.UseQuartzHostedService(stdSchedulerFactoryOptions);
             return re.Services;
         }
+    
+        private static bool _quartzHostedServiceIsAdded = false;
+        /// <summary>
+        ///  Must be call after Host.CreateDefaultBuilder(args).ConfigureWebHostDefaults()
+        /// </summary>
+        /// <param name="builder"></param>
+        /// <seealso cref="https://github.com/aspnet/AspNetCore.Docs/issues/14381"/>
+        ///  <seealso cref="https://github.com/aspnet/AspNetCore/pull/11575"/>
```

Comment:
@mukmyash   WebHost in ASP.NET Core 3 startup order is break!
  QuartzHostedService  tasks  must before IApplicationStartup notified!

Reply:
I understent this... 
but i can't understent, why TargetFramework can't stay netstandard?

I expected next changes:
_QuartzHostedService.csproj_
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
....
  </PropertyGroup>

  <ItemGroup>
    .......
    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="3.0.0" />
  </ItemGroup>

</Project>

```

_IServiceCollectionExtensions.cs_
```
public static IHostBuilder ConfigureQuartzHost(this IHostBuilder builder)
        {
            _quartzHostedServiceIsAdded = true;
            return builder.ConfigureServices((ctx, services) => services.AddHostedService<QuartzHostedService>());
        }
```

_***I'm sorry, if my text seemed rough... my english is bad..._

Reply:
The Microsoft.AspNetCore.Http.Abstractions 's version is too low. can't use to .Net Core 3.0

Reply:
Ouuu .. I did't know, that new versions of Microsoft.AspNetCore.Http.Abstractions is not publish to nuget.

https://stackoverflow.com/questions/57760356/how-do-i-use-asp-net-core-3-0-types-from-a-library-project-for-shared-controller

I approve this changes now.
But i will divide this package on 2 in future.